src/mapping/RadialBasisFctMapping.hpp

```diff
@@ -220,15 +255,35 @@ void RadialBasisFctMapping<RADIAL_BASIS_FUNCTION_T>::mapConservative(DataID inpu
     // Construct Eigen vectors
     Eigen::Map<Eigen::VectorXd> inputValues(globalInValues.data(), globalInValues.size());
     Eigen::VectorXd             outputValues((this->output()->getGlobalNumberOfVertices()) * valueDim);
-    Eigen::VectorXd             in(_rbfSolver.getEvaluationMatrix().rows()); // rows == outputSize
+
+    Eigen::VectorXd in; // rows == outputSize
+
+    if (_useEigen) {
+      in.resize(_rbfSolver.getEvaluationMatrix().rows()); // rows == outputSize
```

Comment:
That's actually a flaw of the eigen solver. We should rather replace the eval getter by a size getter ( if we want to unify the code here).

Reply:
Sounds good. We could then also think of a common interface class.